docs/pages/icons.vue

```diff
@@ -5,79 +5,114 @@
 
     <PageSection title="Overview" anchor="#overview">
       <p>
-        Kolibri products uses a large number of icons that have a special meaning within our ecosystem. They generally draw from <ExternalLink text="Material Icons" href="https://material.io/resources/icons/" />, but may also include custom icons.
+        Kolibri products use a large number of icons that have a special meaning within our ecosystem. They generally draw from <ExternalLink text="Material Icons" href="https://material.io/resources/icons/" />, but may also include custom icons.
       </p>
       <p>
         Importantly, we reference icons by a token specific to the Kolibri Design System rather than Material's names, in order to make token references more meaningful to designers and developers.
       </p>
+    </PageSection>
+
+    <PageSection title="Size and text" anchor="#size-text">
+      <p>
+        Icons should be scaled relative to their surrounding text size. Use the <code>KLabeledIcon</code> component to automatically set a consistent sizing and spacing with associated text:
+      </p>
+      <h3>
+        <KLabeledIcon icon="lesson" label="Header 3" />
+      </h3>
+      <h4>
+        <KLabeledIcon icon="lesson" label="Header 4" />
+      </h4>
+      <h5>
+        <KLabeledIcon icon="lesson" label="Header 5" />
+      </h5>
       <p>
-        Note that many of these icons also are used with  <InternalLink href="colors" text="conventional colors" />
-        in the design system.
+        <KLabeledIcon icon="lesson" label="Paragraph" />
       </p>
+    </PageSection>
 
+    <PageSection title="Color" anchor="#color">
+      <p>
+        When inline with text, icons should be the same color as the text.
```

Comment:
This isn't always true. I'm thinking about the icons in Coach Reports for help needed, completed, etc. That's the only spot I can think of it differing in Kolibri.